db-async-common/src/main/java/com/github/jasync/sql/db/util/NettyUtils.kt

```diff
@@ -41,6 +49,38 @@ object NettyUtils {
         else -> NioSocketChannel::class.java
     }
 
+    fun createSslContext(sslConfiguration: SSLConfiguration): SslContext {
```

Comment:
All the changes in this file just pull out the logic from the postgres driver because it's the exact same for MySQL SSL